src/Migrate.php

```diff
@@ -120,6 +158,70 @@ private function restoreDestinationProject(array $restoreCredentials): void
         $this->logger->info('Current project restored');
     }
 
+    private function migrateSecrets(): void
+    {
+        $this->logger->info('Migrating secrets in configurations');
+
+        $sourceClient = $this->createSourceClient();
+
+        $sourceDevBranches = new DevBranches($sourceClient);
+        $sourceBranches = $sourceDevBranches->listBranches();
+        $defaultSourceBranch = current(array_filter($sourceBranches, fn($b) => $b['isDefault'] === true));
+
+        $sourceComponentsApi = new Components($sourceClient);
+        $components = $sourceComponentsApi->listComponents();
+        if (!$components) {
+            $this->logger->info('There are no components to migrate.');
+            return;
+        }
+
+        $encryptionApiUrl = $sourceClient->getServiceUrl('encryption');
+        $migrations = $this->createMigrationsClient($encryptionApiUrl);
+
+        foreach ($components as $component) {
+            if (in_array($component['id'], self::OBSOLETE_COMPONENTS, true)) {
+                continue;
+            }
+
+            foreach ($component['configurations'] as $config) {
+                $response = $migrations
+                    ->migrateConfiguration(
+                        $this->sourceProjectToken,
+                        Utils::getStackFromProjectUrl($this->destinationProjectUrl),
+                        $this->destinationProjectToken,
+                        $component['id'],
+                        $config['id'],
+                        (string) $defaultSourceBranch['id'],
+                    );
+
+                $this->logger->debug($response['message']);
+            }
+        }
+
+        $this->logger->info('Secrets in configurations have been migrated.');
```

Comment:
to je trošku misleading - reálně je to něco jako "All configurations have been migrated" :)

Reply:
Dal jsem pryč.